<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基于物联网的温室大棚智慧管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/prefix1/apple.png</normaloff>:/new/prefix1/apple.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>SerialPort</string>
   </property>
   <widget class="QLabel" name="SerialStatus">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Disconnected!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Port">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="text">
     <string>COM5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SerialClose">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SerialConnect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>PortName</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/niao.png);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Display">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>451</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Status Information</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Temperature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Humidity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TempValue">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="HumiValue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>％</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>lux</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Light:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LightValue">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="LEDStatus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>LED:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LEDControl">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LCD">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>19</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/Temp.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/water.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/led.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/woniu.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_9" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/ye2.png);</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>721</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>Temp</string>
    </attribute>
    <widget class="QPushButton" name="BtTempView">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>420</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QWidget" name="TempView" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>721</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>51</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/hua.png);</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>400</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/maozhua.png);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>Humid</string>
    </attribute>
    <widget class="QPushButton" name="BtHumidView">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>420</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QWidget" name="HumidView" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>721</width>
       <height>421</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_8" native="true">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>60</y>
        <width>61</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/maozhua.png);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_10" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>51</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/hua.png);</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_13" native="true">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>400</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/maozhua.png);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Light</string>
    </attribute>
    <widget class="QPushButton" name="BtLightView">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>420</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QWidget" name="LightView" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>721</width>
       <height>421</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_11" native="true">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>50</y>
        <width>61</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/maozhua.png);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_12" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>51</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/hua.png);</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_14" native="true">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>400</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/maozhua.png);</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
